scripts/bootloader/generate_zip.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import argparse
+import json
+import time
+from zipfile import ZipFile
```

Comment:
Is this a new dependency?

Reply:
If it is I would prefer if we use a pre-installed zip dependency. E.g.

https://stackoverflow.com/a/7052049/1134134

Reply:
Its a python builtin, so no new dependency

Reply:
comment